packages/node/src/lib/topics/topics.spec.ts

```diff
@@ -88,7 +88,7 @@ describe('Novu Node.js package - Topics class', () => {
 
     const mockedResponse = {
       totalCount: 1,
-      page: 0,
+      page: 1,
```

Comment:
Changes in this file are to adapt to the changes in the API even if they are mocks in the tests, so any future reader is not misled.